subsys/bluetooth/audio/vocs.c

```diff
@@ -19,13 +19,43 @@
 
 #include "audio_internal.h"
 #include "vocs_internal.h"
+#include "zephyr/bluetooth/audio/audio.h"
 
 #define LOG_LEVEL CONFIG_BT_VOCS_LOG_LEVEL
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_vocs);
 
 #define VALID_VOCS_OPCODE(opcode)	((opcode) == BT_VOCS_OPCODE_SET_OFFSET)
 
+#define BT_AUDIO_LOCATION_ALL (BT_AUDIO_LOCATION_FRONT_LEFT | \
```

Comment:
We could define this in `audio.h` similar to `BT_AUDIO_CONTEXT_TYPE_ANY`

Reply:
Done

Reply:
Done